python/main.py

```diff
@@ -0,0 +1,56 @@
+# !/path/to/bin/of/python
+#↑Fill in as appropriate
+
+from abc import ABCMeta, abstractmethod #抽象クラス用
+
+class PartsBehavior(metaclass=ABCMeta):
+    @abstractmethod
+    def getResistance(self):
+        pass
```

Comment:
Use `raise NotImplementedError` is preferable on an abstract class rather than `pass`.